entity-registry/src/main/java/com/linkedin/metadata/aspect/plugins/PluginFactory.java

```diff
@@ -105,6 +85,95 @@ public PluginFactory loadPlugins() {
     return this;
   }
 
+  /**
+   * Memory intensive operation because of the size of the jars. Limit packages, classes scanned,
```

Comment:
Where is the limit being applied?

Reply:
The packages and class names are used in the acceptPackages and acceptClass

> new ClassGraph()
> .acceptPackages(packageNames.stream().distinct().toArray(String[]::new) .acceptClasses(classNames.stream().distinct().toArray(String[]::new))